### DISCUSSION TITLE
PikPak 添加失败 ErrorCode: 3 ,Error: invalid_argument ,ErrorDescription: invalid captcha_sign

### DISCUSSION THREAD

**Discussion by cokebar** [AlistGo/alist#7522] [Q&A]
之前没问题 突然这样子了 可能是更新版本后出问题了 版本号3.39.4
提示报错
ErrorCode: 3 ,Error: invalid_argument ,ErrorDescription: invalid captcha_sign

这个错误代码在 https://github.com/alist-org/alist/pull/7024 中并没有对应的情况 按照这个PR中的说明也未能解决问题

**Comment by dgscyg** [comment]
> 之前没问题 突然这样子了 可能是更新版本后出问题了 版本号3.39.4 提示报错 ErrorCode: 3 ,Error: invalid_argument ,ErrorDescription: invalid captcha_sign
> 
> 这个错误代码在 #7024 中并没有对应的情况 按照这个PR中的说明也未能解决问题

挂载选项选的什么平台

**Reply by dgscyg** [reply]
#7530 应该修复了，等待合并中
